<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Database_Attributes</class>
 <widget class="QWidget" name="Database_Attributes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>539</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>892</width>
    <height>539</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>892</width>
    <height>539</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Attributes Tab</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gui_icons.qrc">
    <normaloff>:/icons/openrpgmaker.png</normaloff>:/icons/openrpgmaker.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">.QFrame
{
	border-width: 1px;
	border-style: solid;
	border-color: palette(window-text);
}

.QFrame&gt;QLabel
{
	border-width: 0px;
	background-color: palette(window-text);
	color: palette(window);
	font-weight: bold;
}

.QFrame&gt;QCheckBox
{
	background-color: palette(window-text);
	border-width: 0px;
	border-style: none;
	color: palette(window);
	spacing: 3px;
	font-weight: bold;
	padding: 0px 0px 0px 3px;
}

.QFrame&gt;QCheckBox::indicator
{
	min-width: 8px;
	max-width: 8px;
	min-height: 8px;
	max-height: 8px;
	width: 8px;
	height: 8px;
	border-width: 2px;
	border-radius: 0px;
	border-style: ridge;
	border-color: palette(mid);
}

.QFrame&gt;QCheckBox::indicator:unchecked
{
	background-color: palette(base);
}

.QFrame&gt;QCheckBox::indicator:checked
{
	background-color: palette(window-text);
}

.QFrame&gt;QCheckBox::indicator:hover
{
	border-color: palette(highlight);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_71">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_112">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_131">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Attributes</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_82" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_114">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QListWidget" name="lwAttributeList">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>214</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>214</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="verticalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bArraySize">
           <property name="text">
            <string>Array Size</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_86" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_160">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_72">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_155">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_132">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_83" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_156">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="leName">
              <property name="maxLength">
               <number>8</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_73">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_157">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_133">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Attribute Type</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_84" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_161">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>3</number>
            </property>
            <item>
             <widget class="QRadioButton" name="rbTypeWeapon">
              <property name="text">
               <string>Weapon</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbTypeMagic">
              <property name="text">
               <string>Magic</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_74">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_158">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_134">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Damage Multiplier</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_85" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_159">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="margin">
             <number>3</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_57">
              <property name="spacing">
               <number>3</number>
              </property>
              <item>
               <widget class="QLabel" name="label_135">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="gui_icons.qrc">:/icons/resistLevelA.png</pixmap>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_52">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QSpinBox" name="sbMultiplierA">
                  <property name="suffix">
                   <string>%</string>
                  </property>
                  <property name="maximum">
                   <number>1000</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="SpinBoxSideButton" name="bMultiplierASpinButton">
                  <property name="minimumSize">
                   <size>
                    <width>13</width>
                    <height>21</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>13</width>
                    <height>21</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset resource="gui_icons.qrc">
                    <normaloff>:/icons/spinBoxSideButton.png</normaloff>:/icons/spinBoxSideButton.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>8</width>
                    <height>16</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_58">
              <property name="spacing">
               <number>3</number>
              </property>
              <item>
               <widget class="QLabel" name="label_136">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="gui_icons.qrc">:/icons/resistLevelB.png</pixmap>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_53">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QSpinBox" name="sbMultiplierB">
                  <property name="suffix">
                   <string>%</string>
                  </property>
                  <property name="maximum">
                   <number>1000</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="SpinBoxSideButton" name="bMultiplierBSpinButton">
                  <property name="minimumSize">
                   <size>
                    <width>13</width>
                    <height>21</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>13</width>
                    <height>21</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset resource="gui_icons.qrc">
                    <normaloff>:/icons/spinBoxSideButton.png</normaloff>:/icons/spinBoxSideButton.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>8</width>
                    <height>16</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_59">
              <property name="spacing">
               <number>3</number>
              </property>
              <item>
               <widget class="QLabel" name="label_137">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="gui_icons.qrc">:/icons/resistLevelC.png</pixmap>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_54">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QSpinBox" name="sbMultiplierC">
                  <property name="suffix">
                   <string>%</string>
                  </property>
                  <property name="maximum">
                   <number>1000</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="SpinBoxSideButton" name="bMultiplierCSpinButton">
                  <property name="minimumSize">
                   <size>
                    <width>13</width>
                    <height>21</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>13</width>
                    <height>21</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset resource="gui_icons.qrc">
                    <normaloff>:/icons/spinBoxSideButton.png</normaloff>:/icons/spinBoxSideButton.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>8</width>
                    <height>16</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_60">
              <property name="spacing">
               <number>3</number>
              </property>
              <item>
               <widget class="QLabel" name="label_138">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="gui_icons.qrc">:/icons/resistLevelD.png</pixmap>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_55">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QSpinBox" name="sbMultiplierD">
                  <property name="suffix">
                   <string>%</string>
                  </property>
                  <property name="maximum">
                   <number>1000</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="SpinBoxSideButton" name="bMultiplierDSpinButton">
                  <property name="minimumSize">
                   <size>
                    <width>13</width>
                    <height>21</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>13</width>
                    <height>21</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset resource="gui_icons.qrc">
                    <normaloff>:/icons/spinBoxSideButton.png</normaloff>:/icons/spinBoxSideButton.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>8</width>
                    <height>16</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_61">
              <property name="spacing">
               <number>3</number>
              </property>
              <item>
               <widget class="QLabel" name="label_139">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="gui_icons.qrc">:/icons/resistLevelE.png</pixmap>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_56">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QSpinBox" name="sbMultiplierE">
                  <property name="suffix">
                   <string>%</string>
                  </property>
                  <property name="maximum">
                   <number>1000</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="SpinBoxSideButton" name="bMultiplierESpinButton">
                  <property name="minimumSize">
                   <size>
                    <width>13</width>
                    <height>21</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>13</width>
                    <height>21</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset resource="gui_icons.qrc">
                    <normaloff>:/icons/spinBoxSideButton.png</normaloff>:/icons/spinBoxSideButton.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>8</width>
                    <height>16</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_75">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_76">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpinBoxSideButton</class>
   <extends>QToolButton</extends>
   <header>spinboxsidebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="gui_icons.qrc"/>
 </resources>
 <connections/>
</ui>
